docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,28 @@
+# Troubleshooting
+If you're reading this, something went wrong.
+Don't worry though, as this is the most thorough guide to help you ~~(and to help us)~~!
```

Comment:
`~~(and to help us)~~` wtf?

Reply:
To help us not having to answer more than once the same questions, I can remove this too.

Reply:
Remove it, looks unprofessional.